features/machine/machine.feature

```diff
@@ -463,8 +463,8 @@ Feature: Machine features testing
       | resource | pv |
     Then the step should succeed
     And the output should contain:
-      | failure-domain.beta.kubernetes.io/zone in [<%= cb.default_zone %>]     |
-      | failure-domain.beta.kubernetes.io/region in [<%= cb.default_region %>] |
```

Comment:
What about backward compatibility, is it changed for 4.7, 4.6?

Reply:
I checked the Automation Tests Results again, seems what I described is not correct. In version 4.8, this case failed on both aws and gcp, but succeeded on azure. In version 4.7, it is successful on aws, gcp and azure.
check result: https://privatebin-it-iso.int.open.paas.redhat.com/?4e1fe62c0be8df68#7Jsah2QigUi1oaBVqzrhmQqo2pi2UL4rvi7G7qdXVDvS

Reply:
Thanks Zhaohua, is this inconsistency an expected behavior?

Reply:
Seems no, based on https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesiozone, Starting in v1.17, this label failure-domain.beta.kubernetes.io is deprecated in favor of topology.kubernetes.io/zone. Not sure why on azure it still is failure-domain.beta.kubernetes.io. Will talk about with storage team.